### PR TITLE
Build with dune

### PR DESCRIPTION
**Pull request by NathanReb**

### CODE REVIEW

File: test/dune [hunk 1]
```diff
@@ -0,0 +1,13 @@
+(alias
```
Comment (bbc2) [inline test/dune]: I think this can be simplified with:

```jbuilder
(test
  (name test_suite)
  (libraries
    enumerators
    oUnit
  )
)
```

**Review by bbc2 — CHANGES_REQUESTED** [review]
Thanks for the migration!

File: src/dune [hunk 2]
```diff
@@ -0,0 +1,5 @@
+(library
+  (name enumerators)
+  (public_name enumerators)
```
Comment (emillon) [inline src/dune]: You can also remove `(name)` if it's equal to the `(public_name)` (requires lang 1.1)

File: test/dune (hunk 1 above)
Comment (NathanReb) [reply test/dune]: hm nice one!

File: src/dune (hunk 2 above)
Comment (NathanReb) [reply src/dune]: also it's `wrapped false` so `name` actually doesn't matter here

**Comment by NathanReb** [comment]
I think it's all fixed!

**Review by bbc2 — APPROVED** [review]

**Pull request merged**